micro-rdk/src/esp32/entry.rs

```diff
@@ -28,117 +28,106 @@ use super::{
 };
 
 use async_io::Timer;
-use futures_lite::Future;
 
 pub async fn serve_web_inner(
     app_config: AppClientConfig,
     _tls_server_config: Esp32TLSServerConfig,
     repr: RobotRepresentation,
     _ip: Ipv4Addr,
     webrtc_certificate: WebRtcCertificate,
-    exec: Esp32Executor<'_>,
+    exec: Esp32Executor,
     max_webrtc_connection: usize,
 ) {
     // TODO(NPM) this is a workaround so that async-io thread has started before we
     // instantiate the Async<TCPStream> for the connection to app.viam.com
     // otherwise there is a chance a race happens and will listen to events before full
     // initialization is done
     let _ = Timer::after(std::time::Duration::from_millis(60)).await;
-    let (mut srv, robot) = {
-        let mut client_connector = Esp32TLS::new_client();
-        let mdns = NoMdns {};
-
-        let (cfg_response, robot) = {
-            let cloned_exec = exec.clone();
-            let conn = client_connector.open_ssl_context(None).unwrap();
-            let conn = Esp32Stream::TLSStream(Box::new(conn));
-            let grpc_client = Box::new(
-                GrpcClient::new(conn, cloned_exec, "https://app.viam.com:443")
-                    .await
-                    .unwrap(),
-            );
-
-            let builder = AppClientBuilder::new(grpc_client, app_config.clone());
-
-            let mut client = builder.build().await.unwrap();
-
-            let (cfg_response, cfg_received_datetime) = client.get_config().await.unwrap();
-
-            let robot = match repr {
-                RobotRepresentation::WithRobot(robot) => Arc::new(Mutex::new(robot)),
-                RobotRepresentation::WithRegistry(registry) => {
-                    log::info!("building robot from config");
-                    let r = match LocalRobot::from_cloud_config(
-                        &cfg_response,
-                        registry,
-                        cfg_received_datetime,
-                    ) {
-                        Ok(robot) => {
-                            if let Some(datetime) = cfg_received_datetime {
-                                let logs = vec![config_log_entry(datetime, None)];
-                                client
-                                    .push_logs(logs)
-                                    .await
-                                    .expect("could not push logs to app");
-                            }
-                            robot
+
```

Comment:
GH diff didn't do a great job here, but from what I can tell there are no substantive changes here, right?

Reply:
no not really :)